services/metadata_service/tests/integration_tests/utils.py

```diff
@@ -209,17 +220,32 @@ async def assert_api_get_response(cli, path: str, status: int = 200, data: objec
         http status code to expect from response
     data : object
         An object to assert the api response against.
+    data_is_unordered_list : bool
+        Data is an unordered list, so ignore ordering when comparing data and response body
     """
     response = await cli.get(path)
 
     assert response.status == status
 
-    if data:
-        body = json.loads(await response.text())
+    body = json.loads(await response.text())
+    if data is None:
+        return
+    if data_is_unordered_list:
+        assert type(data) == list and type(body) == list
```

Comment:
```suggestion
        assert isinstance(data, list) and isinstance(body, list)
```
I think `isinstance` is a lil more idiomatic and more proof if the value is some kind of a list derivative.